instance/models/database_server.py

```diff
@@ -120,6 +121,10 @@ class DatabaseServer(ValidateModelMixin, TimeStampedModel):
     class Meta:
         abstract = True
 
+    # Human readable identifier for database instances
+    name = models.CharField(max_length=128, unique=True)
+    description = models.CharField(max_length=128, blank=True)
```

Comment:
@bradenmacdonald Do you have an approach in mind for the refactor? I agree your objections are valid ones, but I'm not sure how we could go at standardizing this differently, without keeping the fields/classes duplicated and having to keep them in sync manually? Having to remember to check the other similar classes elsewhere for consistency is also a form of complexity.